app/controllers/allocation_staff_controller.rb

```diff
@@ -86,4 +86,16 @@ def load_prisoner_via_prisoner_id
   def coworking?
     params[:coworking].present? && params[:coworking] == 'true'
   end
+
+  def compare_success_route
```

Comment:
This will make more sense later on, as there will be a new "reallocations" controller inheriting from this one and these routes need to be customisable.